packages/playwright-core/src/common/userAgent.ts

```diff
@@ -44,14 +43,11 @@ function determineUserAgent(): string {
     osIdentifier = 'macOS';
     osVersion = `${version[0]}.${version[1]}`;
   } else if (process.platform === 'linux') {
-    try {
-      // List of /etc/os-release values for different distributions could be
-      // found here: https://gist.github.com/aslushnikov/8ceddb8288e4cf9db3039c02e0f4fb75
-      const osReleaseText = fs.readFileSync('/etc/os-release', 'utf8');
-      const fields = parseOSReleaseText(osReleaseText);
-      osIdentifier = fields.get('id') || 'unknown';
-      osVersion = fields.get('version_id') || 'unknown';
-    } catch (e) {
+    const distroInfo = getLinuxDistributionInfoSync();
+    if (distroInfo) {
+      osIdentifier = distroInfo.id || 'unknown';
```

Comment:
Default to 'linux' instead, similar to the `else` branch?

Reply:
sure